hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This patch improves the performance of unique collections such as
+:func:`hypothesis.strategies.sets` when the elements are drawn from a
+:func:`hypothesis.strategies.sampled_from` strategy (:issue:`1885`).
+There are similar improvements in ``sampled_from(...).filter(...)``.
```

Comment:
These two changes are conceptually similar, but don't seem to be linked in any way at the implementation level.

Maybe that's a sign that they should be two separate PRs.